widget/widgetAssets/js/iocExtractionConfigService.service.js

```diff
@@ -62,10 +64,40 @@
             "key",
             "jSONValue"
           ]
+        },
+        globalVariablesToKeyStoreMapping: {
+          "sfsp-excludelist-ips": ["Excludelist_IPs", "IP Address", true],
```

Comment:
exclude ioc global variables - don't delete them after migration

another suggestion came from Abhishek: read/write of these ioc global variables/keystore can be done using some dynamic expressions so that we can create a abstraction layer and use them in PB. With this user don't need change their playbooks evenif changes underneath storing mechanism. please capture this as one the ask and check its feasibility.

Reply:
Sure

Reply:
Global variable deletion logic has been removed

Reply:
> another suggestion came from Abhishek: read/write of these ioc global variables/keystore can be done using some dynamic expressions so that we can create a abstraction layer and use them in PB. With this user don't need change their playbooks evenif changes underneath storing mechanism. please capture this as one the ask and check its feasibility.

will explore the implementation  feasibility of this